admin/spec/components/previews/solidus_admin/ui/pagination/component_preview.rb

```diff
@@ -4,4 +4,27 @@
 class SolidusAdmin::UI::Pagination::ComponentPreview < ViewComponent::Preview
   include SolidusAdmin::Preview
 
+  def overview
+    render_with_template(
+      locals: {
+        page: page_proc
+      }
+    )
+  end
+
+  # @param left toggle
+  # @param right toggle
+  def playground(left: false, right: false)
+    render current_component.new(
+      page: page_proc.call(left, right)
+    )
+  end
+
+  private
+
+  def page_proc
+    lambda { |left, right|
+      Struct.new(:records, :number, :next_param, :first?, :last?).new(nil, 1, '#', !left, !right)
```

Comment:
Probably we don't need to stub `:records`, as we're not using it in the component, but I might be missing something.